src/book/03-guides/01-main-codebase/02-maintaining-subsystems.mdx

```diff
@@ -10,6 +10,17 @@ authors:
 
 This page details how to maintain various subsystems within the main codebase.
 
+## Odometry
+
+### Webots Live Error Tracking
+
+The `nugus_controller` in Webots has functionality to send live odometry ground truth data to the robot. To use this
+
+1. [Set up Webots and compile the NUWebots code](/guides/tools/webots-setup) (you do not need the RoboCup set up).
+2. Run the `kid.wbt` world in Webots.
+3. Run `webots_keyboardwalk` in NUbots. [See the Getting Started page to set up the NUbots codebase](/guides/main/getting-started).
```

Comment:
This is now `webots/keyboardwalk` no?

Reply:
Not yet, not sure when I'll get around to testing the folder roles functionality on the robot.